Result of `nixpkgs-review pr 325590` run on aarch64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package failed to build:</summary>
  <ul>
    <li>prusa-slicer</li>
  </ul>
</details>

The error looks to be the same as with 2.7.4 currently so not introduced by this PR:
<details>
<summary>no template named 'binary_function' in namespace 'boost::functional::detail'; did you mean '__binary_function'?</summary>
<pre>
error: builder for '/nix/store/kg16943n1fm1yap9mh01hvma41ba6q1c-prusa-slicer-2.8.0.drv' failed with exit code 2;
       last 25 log lines:
       > include/boost/functional.hpp:45:24: note: '__unary_function' declared here
       >             using std::unary_function;
       >                        ^
       > include/boost/functional.hpp:487:68: error: no template named 'binary_function' in namespace 'boost::functional::detail'; did you mean '__binary_function'?
       >     class const_mem_fun1_ref_t : public boost::functional::detail::binary_function<const T&, A, S>
       >                                         ~~~~~~~~~~~~~~~~~~~~~~~~~~~^
       > include/boost/functional.hpp:46:24: note: '__binary_function' declared here
       >             using std::binary_function;
       >                        ^
       > include/boost/functional.hpp:533:73: error: no template named 'unary_function' in namespace 'boost::functional::detail'; did you mean '__unary_function'?
       >     class pointer_to_unary_function : public boost::functional::detail::unary_function<Arg,Result>
       >                                              ~~~~~~~~~~~~~~~~~~~~~~~~~~~^
       > include/boost/functional.hpp:45:24: note: '__unary_function' declared here
       >             using std::unary_function;
       >                        ^
       > include/boost/functional.hpp:557:74: error: no template named 'binary_function' in namespace 'boost::functional::detail'; did you mean '__binary_function'?
       >     class pointer_to_binary_function : public boost::functional::detail::binary_function<Arg1,Arg2,Result>
       >                                               ~~~~~~~~~~~~~~~~~~~~~~~~~~~^
       > include/boost/functional.hpp:46:24: note: '__binary_function' declared here
       >             using std::binary_function;
       >                        ^
       > 16 errors generated.
       > make[2]: *** [src/libslic3r/CMakeFiles/libslic3r.dir/build.make:77: src/libslic3r/CMakeFiles/libslic3r.dir/cmake_pch.hxx.pch] Error 1
       > make[1]: *** [CMakeFiles/Makefile2:1078: src/libslic3r/CMakeFiles/libslic3r.dir/all] Error 2
       > make: *** [Makefile:146: all] Error 2
<pre>
</details>

--- a/pkgs/applications/misc/prusa-slicer/default.nix
+++ b/pkgs/applications/misc/prusa-slicer/default.nix
@@ -2,6 +2,7 @@
 , lib
 , binutils
 , fetchFromGitHub
+, fetchpatch
 , cmake
 , pkg-config
 , wrapGAppsHook3
@@ -15,6 +16,7 @@
 , expat
 , glew
 , glib
+, glib-networking
 , gmp
 , gtk3
 , hicolor-icon-theme
@@ -64,15 +66,27 @@ let
   });
   openvdb_tbb_2021_8 = openvdb.override { tbb = tbb_2021_11; };
   wxGTK-override' = if wxGTK-override == null then wxGTK-prusa else wxGTK-override;
+
+  patches = [
+    (fetchpatch {
+      url = "https://raw.githubusercontent.com/gentoo/gentoo/master/media-gfx/prusaslicer/files/prusaslicer-2.8.0-missing-includes.patch";
+      hash = "sha256-/R9jv9zSP1lDW6IltZ8V06xyLdxfaYrk3zD6JRFUxHg=";
+    })
+    (fetchpatch {
+      url = "https://raw.githubusercontent.com/gentoo/gentoo/master/media-gfx/prusaslicer/files/prusaslicer-2.8.0-fixed-linking.patch";
+      hash = "sha256-G1JNdVH+goBelag9aX0NctHFVqtoYFnqjwK/43FVgvM=";
+    })
+  ];
 in
 stdenv.mkDerivation (finalAttrs: {
   pname = "prusa-slicer";
-  version = "2.7.4";
+  version = "2.8.0";
+  inherit patches;
 
   src = fetchFromGitHub {
     owner = "prusa3d";
     repo = "PrusaSlicer";
-    hash = "sha256-g2I2l6i/8p8werDs4mDI/lGeDQsma4WSB9vT6OB2LGg=";
+    hash = "sha256-A/uxNIEXCchLw3t5erWdhqFAeh6nudcMfASi+RoJkFg=";
     rev = "version_${finalAttrs.version}";
   };
 
@@ -93,6 +107,7 @@ stdenv.mkDerivation (finalAttrs: {
     expat
     glew
     glib
+    glib-networking
     gmp
     gtk3
     hicolor-icon-theme
@@ -127,9 +142,6 @@ stdenv.mkDerivation (finalAttrs: {
 
   # prusa-slicer uses dlopen on `libudev.so` at runtime
   NIX_LDFLAGS = lib.optionalString withSystemd "-ludev";
-
-  # FIXME: remove in 2.8.0
-  NIX_CFLAGS_COMPILE = "-Wno-enum-constexpr-conversion";
 
   prePatch = ''
     # Since version 2.5.0 of nlopt we need to link to libnlopt, as libnlopt_cxx

